src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java

```diff
@@ -1772,6 +1772,7 @@ default MessagePaginationAction getIterableHistory()
         Checks.notNull(unicode, "Provided Unicode");
         unicode = unicode.trim();
         Checks.notEmpty(unicode, "Provided Unicode");
+        // TODO Maybe checking for Checks.noWhitespace(unicode, "Unicode emoji") instead (in terms of consistency / making it uniform)
```

Comment:
No this is intended, to allow things like `addReaction("U+AABB U+CCDD")`

Reply:
For adding multiple reactions at once? Thought you don't allow that. Or is it like 2 concatenated unicodes `\uD83D\uDCFA` for adding like 📺?

Reply:
The latter

Reply:
So to stay on this concrete example:
Both `addReaction("U+1F4FA")` and `addReaction("U+D83D U+DCFA")` would add the reaction correctly? Because I never saw the codepoints used like that.